relationships.md

```diff
@@ -139,6 +146,23 @@ Error codes:
 * 404 Not found - if the relationships doesn't exist
 * 422 Unprocessable Entity - if the item doesn't exist, or if the amount of items is not 1
 
+## Updating the metadata of a relationship
+
+**PUT /api/core/relationships/<:id>**
+
+Update the name variant or the place:
+```
+curl -i -X PUT 'https://dspace7-entities.atmire.com/rest/api/core/relationships/891' -H 'Authorization: Bearer eyJhbGciO…' -H "Content-Type:application/json" --data '{ "leftPlace": 1, "leftWardLabel": "Name variant 1", "rightPlace": 1, "rightWardLabel": null }'
+```
+
+Omitted properties will not be updated. Removing a name variant would require setting it to null.
```

Comment:
Unless I'm misunderstanding, I think this statement is inaccurate: "Omitted properties will not be updated."  A `PUT` request is supposed to be a full *replacement*, per our REST Contract README: https://github.com/DSpace/Rest7Contract#on-single-resource-endpoints   If you wanted to do a partial update, we'd need to define a `PATCH` request for that.

Reply:
you're right, I've updated this in the rest contract